docs/mevblocker/users-and-integrators/Users/available-endpoints.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+---
+
+# MEV Blocker Endpoints 
+As a MEV Blocker user, you can choose across three different endpoints based on your transaction needs.
+
+[Fast](https://rpc.mevblocker.io/fast) - focused on getting your transaction included onchain as fast as possible while forwarding you a rebate.
+
+[No Revert](https://rpc.mevblocker.io/noreverts) - focused on getting your transaction included onchain without any possibilities of reverting. In case your transaction is no longer valid it will stop being broadcasted to builders at no cost for the user.
```

Comment:
```suggestion
[No Revert](https://rpc.mevblocker.io/noreverts) - focused on getting your transaction included on-chain without any possibilities of reverting. In case your transaction is no longer valid it will stop being broadcasted to builders at no cost for the user.
```